src/app/screens/form-register/form-register.component.html

```diff
@@ -3,27 +3,41 @@
     <form [formGroup]='registerForm' (ngSubmit)='addPost()'>
       <div class="section-input">
         <label class="text-label">Nombre</label>
-        <input class="decoration-input" type="text" formControlName="firstName" />
-        <span class="message-alert" *ngIf="!registerForm.controls.firstName.valid && registerForm.controls.firstName.touched">{{ messageAlert }}</span>
+        <input class="decoration-input" type="text" formControlName="first_name" />
+        <span class="message-alert" *ngIf="!registerForm.controls.first_name.valid && registerForm.controls.first_name.touched">{{ messageAlert }}</span>
       </div>
       <div class="section-input">
         <label class="text-label">Apellido</label>
-        <input class="decoration-input" type="text" formControlName="lastName" />
-        <span class="message-alert" *ngIf="!registerForm.controls.lastName.valid && registerForm.controls.lastName.touched">{{ messageAlert }}</span>
+        <input class="decoration-input" type="text" formControlName="last_name" />
+        <span class="message-alert" *ngIf="!registerForm.controls.last_name.valid && registerForm.controls.last_name.touched">{{ messageAlert }}</span>
       </div>
       <div class="section-input">
         <label v>Correo</label>
         <input class="decoration-input" type="text" formControlName="email" />
         <span class="message-alert"s *ngIf="!registerForm.controls.email.valid && registerForm.controls.email.touched">{{ messageAlert }}</span>      
```

Comment:
Try to user typescript getters to minimize logic in the html https://ultimatecourses.com/blog/typescript-setters-getter